marimo/_output/data/data.py

```diff
@@ -33,3 +43,45 @@ def image(data: bytes, ext: str = "png") -> VirtualFile:
     item = VirtualFileLifecycleItem(ext=ext, buffer=data)
     get_context().cell_lifecycle_registry.add(item)
     return item.virtual_file
+
+
+def any_data(data: Union[str, bytes, io.BytesIO], ext: str) -> VirtualFile:
```

Comment:
nite: `data`'s type should be `Optional[Union[...]]` since `data` can be `None`

Reply:
Looks like it should also include `io.FileIO`? What types are you trying to cover?

Reply:
ill update to include both of those. ideally i cover all common types. not sure what will exist, but str and bytes cover most of what i've hit. ill also add `pd` here in the future as well